diff --git a/node_modules/react-qr-reader/lib/index.js b/node_modules/react-qr-reader/lib/index.js
index fb06076..bc975ed 100644
--- a/node_modules/react-qr-reader/lib/index.js
+++ b/node_modules/react-qr-reader/lib/index.js
@@ -432,7 +432,26 @@ module.exports = (_temp = _class = function (_Component) {
         React.createElement(
           'section',
           { style: containerStyle },
-          !legacyMode && showViewFinder ? React.createElement('div', { style: viewFinderStyle }) : null,
+          !legacyMode && showViewFinder ?React.createElement(
+            'div',
+            {
+              style: { zIndex: 50 },
+              className:
+                'h-full   w-full absolute right-0 left-0  top-0 bottom-0 flex items-center justify-center',
+            },
+            React.createElement(
+              'div',
+              {
+                className: 'h-3/6  z-50 w-6/12  qr-scanner',
+              },
+              React.createElement(
+                'div',
+                { className: 'box' },
+                React.createElement('div', { class: 'line' }),
+                React.createElement('div', { class: 'angle' })
+              )
+            )
+          ) : null,
           legacyMode ? React.createElement('input', {
             style: hiddenStyle,
             type: 'file',
